.changeset/strong-owls-sleep.md

```diff
@@ -0,0 +1,29 @@
+---
+'@itwin/itwinui-css': major
+---
+
+The `iui-tabs` element has changed from `<ul>` to `<div>`.
+`iui-tabs` children are no longer wrapped in an `<li>` element. 
+The `iui-tab-label` wrapper class has been removed. 
+The `iui-tab-label` class now applies to the `<span>` which holds the tab's label. 
+`iui-tab-description` is now a `<span>` element.
```

Comment:
nit: A list would be better

Reply:
Updated changesets to be in separate files.